api/src/models/publication.model.js

```diff
@@ -52,8 +52,6 @@ const publicationSchema = new mongoose.Schema(
       },
       categoryTitle: {
         type: String,
-        required: true,
```

Comment:
This was removed because the user might not have given a category title when adding their publications to google scholar